src/commands/xread.json

```diff
@@ -8,8 +8,7 @@
         "get_keys_function": "xreadGetKeys",
         "command_flags": [
             "BLOCKING",
-            "READONLY",
-            "BLOCKING"
```

Comment:
i don't think we should remove the BLOCKING flag.
i think it is set on any command that has the **potential** to be blocking.

Reply:
it's a duplicated one, look at line 10